assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -23,9 +22,10 @@ const Background = styled.div`
 
 	display: grid;
 	grid:
-    ".   meta       .        ." 1fr
-    ".   page       addpage  ." ${ PAGE_HEIGHT }px
-    ".   carrousel  .        ." 1fr
+    ".     .          .        ." 1fr
```

Comment:
This space is now more like 36px.

Reply:
```suggestion
    ".     .          .        ." 36px
```

Reply:
36px? Where did you get that? It see 12/25 px between the top of the page canvas and the bottom of the publish button. Or 48/77 px between the top of the page canvas to the top of the editor overall. But I think this is best left for future responsive layout efforts post #4110.